sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/OpenAIClientTestBase.java

```diff
@@ -146,41 +135,86 @@ private List<ChatMessage> getChatMessages() {
         return chatMessages;
     }
 
-    static void assertChatCompletions(int[] indexArray, ChatRole[] chatRoleArray, ChatCompletions actual) {
-        List<ChatChoice> choices = actual.getChoices();
-        assertNotNull(choices);
-        assertTrue(choices.size() > 0);
-        assertChatChoices(indexArray, chatRoleArray, choices);
+    static void assertCompletions(int choicesPerPrompt, Completions actual) {
+        assertCompletions(choicesPerPrompt, "stop", actual);
```

Comment:
I saw you have tests with finish reasons "stop" and "length". How about cases for "content_filter" or "null"?
I actually don't know how useful those would be, I only found them when searching for [`finish_reason` values](https://platform.openai.com/docs/guides/chat/response-format) (which I only could only find with an advanced search engine query, does the OpenAI documentation really not have a "search" feature?)

Reply:
Regarding `null` I think that the only reason this is nullable is because of streaming. Although, I am not entirely sure. I think that it could be useful once we have live testing to assert every possible enum value.